gocumber.go

```diff
@@ -38,6 +38,19 @@ func ColumnMap(table Table) map[string]string {
 	return result
 }
 
+func RowMap(table Table) map[string]string {
+	result := make(map[string]string)
+
+	header := table.Rows()[0]
+	for _, row := range table.Rows()[1:] {
+		for i, key := range header {
+			result[key] = row[i]
+		}
+	}
```

Comment:
Is this going to work for when you have more than one row in the table? it seems like result[key] is going to keep getting overwritten until the last row?